lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -82,19 +83,69 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
       selected.forEach((option, index) => {
         option._selectedness = index === selected.length - 1;
       });
+      this._selectedOption = selected[selected.length - 1];
     }
   }
 
+  _updateSelected(child) {
+    if (this.hasAttributeNS(null, "multiple")) {
+      return;
+    }
+    if (child.hasAttributeNS(null, "disabled")) {
+      return;
+    }
+    const optArr = this._toOptionArray(child);
+    if (!optArr.length) {
```

Comment:
Style nit: prefer numeric comparisons for `length`, so `if (optArray.length === 0)`, and below `if (selectedOptions.length > 0)`. (I realize there is some code elsewhere in this file that does not conform to that...)

Reply:
Done